src/mixpanelAnalytics.ts

```diff
@@ -0,0 +1,52 @@
+import Assignment from './assignment';
+
+type EventProperties = {
+  TTVisitorID: string;
+  SplitName: string;
+  SplitVariant: string | null;
+  SplitContext?: string;
+};
+
+declare global {
+  interface Window {
+    mixpanel: {
+      track(eventName: string, properties: EventProperties, callback: (value: boolean) => void): void;
+      identify(id: string): void;
+      alias(id: string): void;
+    };
+  }
+}
+
+export interface AnalyticsProvider {
```

Comment:
move to Session or its own file? thinking through someone using this interface, it would be odd to have to import `mixpanelAnalytics`

Reply:
This is exported from the root file, `testTrack.js`, but I understand what you mean. I'll move it under its own definition file.